docs/components/modeler/bpmn/script-tasks/script-tasks.md

```diff
@@ -25,13 +25,13 @@ Jobs for script tasks are not processed by Zeebe itself. To process them, provid
 
 ## Defining a task
 
-A script task must define a [job type](/components/modeler/bpmn/service-tasks/service-tasks.md#task-definition). It specifies
+A script task must define a [job type](/components/modeler/bpmn/service-tasks/service-tasks.md#task-definition) the same way as a service task does. It specifies
 the type of job workers should subscribe to (e.g. `script`).
 
 Use [task headers](/components/modeler/bpmn/service-tasks/service-tasks.md#task-headers) to pass static parameters to the job
-worker (e.g. the script to evaluate).
+worker (e.g. the script to evaluate). The community extension [Zeebe Script Worker](https://github.com/camunda-community-hub/zeebe-script-worker) requires certain attributes to be set in the task headers.
 
-Define [variable mappings](/components/modeler/bpmn/service-tasks/service-tasks.md#variable-mappings) to transform the
+Define [variable mappings](/components/concepts/variables.md#inputoutput-variable-mappings) to transform the
```

Comment:
❌  I have the same concern here as I mentioned [here](https://github.com/camunda-cloud/camunda-cloud-documentation/pull/602#discussion_r806036713). Please reconsider.

Reply:
I reconsidered